src/Installers/Windows/Wix.props

```diff
@@ -14,7 +18,7 @@
   <!-- Additional parameters for Light/Candle-->
   <PropertyGroup>
     <LinkerAdditionalOptions>-fv</LinkerAdditionalOptions>
-    <SuppressIces>ICE61</SuppressIces>
+    <SuppressIces>ICE61;ICE03</SuppressIces>
```

Comment:
Personal preference would be to scope this to the projects that need it instead of applying this to all windows installer projects. ICE03 covers a wide range of validation problems that would be good to catch. We should also look at fixing ICE61 errors.